src/main/java/frc/robot/Robot.java

```diff
@@ -7,8 +7,8 @@
 
 package frc.robot;
 
-import com.chopshop166.chopshoplib.DashboardUtils;
 import com.chopshop166.chopshoplib.RobotUtils;
+import com.chopshop166.chopshoplib.commands.CommandRobot;
```

Comment:
Inherit from `CommandRobot` instead of `TimedRobot` on line 52, then on line 62 you can remove the `CommandRobot.` part of it